projects/noloan/common/src/main/proto/Court.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
```

Comment:
What about removing `projects/noloan/common/src/main/proto/Court.proto` and related files?
Does the code use the new files?